content/docs/about/overview.md

```diff
@@ -7,9 +7,71 @@ Learn more about {{< reuse "docs/snippets/product-name.md" >}}, its architecture
 
 ## About kgateway
 
-{{< reuse "docs/snippets/product-name-caps.md" >}} is a feature-rich, fast, and flexible Kubernetes-native ingress controller and next-generation API gateway that is built on top of [Envoy proxy](https://www.envoyproxy.io/) and the {{< reuse "docs/snippets/k8s-gateway-api-name.md" >}}. An API Gateway is a reverse proxy that serves as a security barrier between your clients and the microservices that make up your app. In order to access a microservice, all clients must send a request to the API Gateway. The API Gateway then verifies and routes the request to the microservice.
+{{< reuse "docs/snippets/product-name-caps.md" >}} is a feature-rich, fast, and flexible Kubernetes-native [ingress controller](#what-is-an-ingress) and next-generation [API gateway](#what-is-an-api-gateway) that is built on top of [Envoy proxy](https://www.envoyproxy.io/) and the [{{< reuse "docs/snippets/k8s-gateway-api-name.md" >}}](#what-is-the-kubernetes-gateway-api). 
 
-{{< reuse "docs/snippets/product-name-caps.md" >}} is fully conformant with the {{< reuse "docs/snippets/k8s-gateway-api-name.md" >}} and extends its functionality with custom Gateway APIs, such as RoutePolicies, ListenerPolicies, or Backends. These resources help to centrally configure advanced traffic management, security, and resiliency rules for an HTTPRoute or Gateway listener.
+### What is an ingress?
+
+An ingress, edge router, or application gateway, is a service that is accessible to the Internet, and routes traffic to services running inside a Kubernetes environment. Traditionally, this service was managed by a Kubernetes object also called an `Ingress`, and as such “ingress” in a Kubernetes context normally refers to an operator configuration where a control plane configures a hardware or software proxy server. The traffic can include API traffic, but can also include general traffic like web pages and images.
+
+Examples of Kubernetes ingresses include ingress-nginx and Contour.
```

Comment:
Just to use the product name vs. the GitHub project name
```suggestion
Examples of Kubernetes ingresses include Ingress NGINX Controller and Contour.
```

Reply:
I'm going to keep this one as-is, because the community tends to refer to it as 'ingress-nginx' (see https://kubernetes.io/blog/2025/03/24/ingress-nginx-cve-2025-1974/ for example) and it can be confused with F5's [NGINX Ingress Controller](https://www.f5.com/products/nginx/nginx-ingress-controller).